src/integration-test/tests/citizen/defence/pages/defendant-claim-details.ts

```diff
@@ -17,6 +17,6 @@ export class DefendantClaimDetails {
     I.see(claimData.total)
     I.see(claimData.reason)
     I.click(fields.amountBreakdown)
+    I.click('Download claim')
```

Comment:
This will probably pass when PDF generation has failed when this is last check in the test. Maybe we should at least check that we remained on the same page and no error page was displayed.